pdns/recursordist/settings/table.py

```diff
@@ -2981,11 +2981,11 @@
         'name' : 'udp_source_port_avoid',
         'section' : 'outgoing',
         'type' : LType.ListStrings,
-        'default' : '11211',
+        'default' : '11211,4791',
         'help' : 'List of comma separated UDP port number to avoid',
```

Comment:
I know it's not part of your changes, but while you're in this file, could you please fix the typo here?
```suggestion
        'help' : 'List of comma separated UDP port numbers to avoid',
```

Reply:
i done this